webui/tests/cypress/integration/02-navigation.spec.ts

```diff
@@ -9,6 +12,10 @@ describe('Navigation', () => {
 
   describe('paths', () => {
 
+    beforeEach(() => {
+      cy.restoreLocalStorageCache([ STORAGE_KEY_AUTH ]);
+    });
+
```

Comment:
**suggestion:** move this outside of `describe('paths', ...)` so you don't have to duplicate `restoreLocalStorageCache` for `describe('side menu buttons', ...)`?

Reply:
no need anymore https://github.com/determined-ai/determined/pull/1539/commits/69aaa838349fe2e27f7d966d9822c8e36d40bd9e